### ISSUE TITLE
请假一个问题，使用mp_size=2时的loss应该怎么写

### ISSUE THREAD

**Issue by kunden0612** [THUDM/SwissArmyTransformer#131]
logits, *mems = model(inputs_ids, position_ids, attention_mask)
    # print(logits.shape)
    loss_func = CrossEntropyLoss(ignore_index=-100)
    loss = loss_func(logits.view(-1, logits.size(-1)).float(), labels.view(-1))``
我是这样写的loss计算方式，会出现一个

**Comment by 1049451037** [comment]
是不是你forward的时候传了`parallel_output=True`：https://github.com/THUDM/SwissArmyTransformer/blob/main/sat/transformer_defaults.py#L146

导致输出还没有聚合
